### ISSUE TITLE
Implement UI to invite a user to a role in a unit

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/frontend#355]
# The UX - flow:
This is available on the [RealTimeBoard](https://realtimeboard.com/app/board/o9J_k0ckGU4=/)
![image](https://user-images.githubusercontent.com/31331637/42254924-c2b00282-7f7b-11e8-8b5c-ce491b10cf1f.png)

# The screens we need to implement/modify:

- Add the edit icon to the overview screen (also relevant for #352)
- Add the design 'Invite users to the unit'

@kiatlim, can you make sure that the relevant screens are available in the Invision too: I couldn't find them there :sweat_smile: 

We should be able to reuse most of what we have done for the 'Invite a user to a case' scenario: invite a user to a case = invite a user to a unit + invite a user to a case.

**Comment by michkwt** [comment]
![unee-t user flow - invite user to the unit](https://user-images.githubusercontent.com/38183796/42365589-81dc195a-8131-11e8-808f-5cad5c1fb224.jpg)

https://invis.io/UPMN8W7298V#/307215602_3-3-0_MVP-All_Units_Copy_10_-2-

**Comment by franck-boullier** [comment]
@michkwt I think that, based on the latest invited user signup flow we discussed yesterday, this last bit is obsolete:

![image](https://user-images.githubusercontent.com/31331637/42366584-1d90879e-8134-11e8-9ba2-2db418f2c9c8.png)

**Comment by michkwt** [comment]
@franck-boullier Yes. That's mistake on my end. I've updated the realtimeboard and here's the updated flow. 
![unee-t user flow - invite user to the unit 1](https://user-images.githubusercontent.com/38183796/42371368-e38941b8-8141-11e8-8a14-c379a5814c0c.jpg)

https://invis.io/UPMN8W7298V#/307215602_3-3-0_MVP-All_Units_Copy_10_-2-
The screens in invision ends after invite sent message screen but the flow can be seen in the image above and on RTB.
